pkg/osinfo/container_scanner.go

```diff
@@ -56,6 +56,8 @@ func ReadOSPackages(layers []string) (
 		cs = newDebianScanner()
 	case OSAlpine, OSWolfi:
 		cs = newAlpineScanner()
+	case OSAmazonLinux, OSFedora, OSRHEL:
```

Comment:
nit for later: we can also add opensuse ! 🦎
(Not actual chameleon emoji, but close :)